src/platform/mt793x/DnssdImpl.h

```diff
@@ -91,6 +90,23 @@ class MdnsContexts
     std::vector<GenericContext *> mContexts;
 };
 
+struct RegisterContext : public GenericContext
+{
+    DnssdPublishCallback callback;
+    std::string mType;
+    std::string mInstanceName;
+    // HostNameRegistrar mHostNameRegistrar;
+
+    RegisterContext(const char * sType, const char * instanceName, DnssdPublishCallback cb, void * cbContext);
+    // virtual ~RegisterContext() { mHostNameRegistrar.Unregister(); }
```

Comment:
Why is this commented-out code here?

Reply:
Oh..., there are some messy code during development, I just removed it.